src/index.html

```diff
@@ -11,7 +11,138 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <!-- <h1>HTML Form</h1> -->
+    <form
+     action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post"
+    >
+      <fieldset class="field">
+        <legend>Personal information</legend>
+
+        <label class="input">
+          Surname:
+          <input type="text"  
+           name="surname" 
+           autocomplete="off"
+           required
+          >
+        </label>
+
+        <label class="input">
+          Name:
+          <input type="text"
+           name="name"
+           autocomplete="off"
+          >
+        </label>
+
+        <label class="input">
+          How old are You?
+          <input type="number"
+           name="age"
+           min="1"
+           max="100"
+           value="12"
+          >
+        </label>
+
+        <label class="input">
+          Full date of birth:
+          <input type="date" name="date">
+        </label>
+
+        <label class="input">
+          I accept the term of the agreement
+          <input type="checkbox" name="agree">
+        </label>
+      </fieldset>
+
+      <fieldset class="field">
+        <legend>Registration</legend>
+
+        <label class="input">
+          E-mail:
+          <input type="email"
+           name="email" 
+          placeholder ="email@example.com"
+          >
+        </label>
+        
+        <label class="input">
+          Password:
+          <input type="password"
+           name="password"
+           minlength="8"
+          >
+        </label>
+      </fieldset>
+
+      <fieldset class="field">
+        <legend>An interesting fact about you!</legend>
+
+        <label class="input">
+        Do you love cats?
+          <input type="radio"
+         name="cats" 
+         value="yes">
+        Yes
+          <input type="radio"
+         name="cats"
+         value="no">
+        No
+        </label>
```

Comment:
```suggestion
      <label class="input">
          Do you love cats?
          <label>
            <input
              type="radio"
              name="cats"
              value="yes"
            >
            Yes
          </label>

          <label>
            <input
              type="radio"
              name="cats"
              value="no"
            >
            No
          </label>
        </label>
```
<img width="319" alt="image" src="https://user-images.githubusercontent.com/102413088/185623929-3f1a11df-a5ab-4ae6-83ba-bab9a3a705f2.png">